colcon_core/extension_point.py

```diff
@@ -6,11 +6,17 @@
 import os
 import traceback
 
+try:
+    from importlib.metadata import distributions
+    from importlib.metadata import EntryPoint
+    from importlib.metadata import entry_points
+except ImportError:
```

Comment:
It will succeed unconditionally as of Python 3.8, actually. I added a comment in the sources, but the biggest cue that we can change something is the entry in the `setup.cfg`, which is conditional from Python < 3.8.